symfony/plugins/orangehrmPimPlugin/lib/dao/EmployeeDao.php

```diff
@@ -884,113 +918,117 @@ public function getEmployeeList($orderField = 'lastName', $orderBy = 'ASC', $inc
-                            ->select('e.empNumber')
-                            ->from('Employee e');
-                
-                if ($excludeTerminatedEmployees) {
-                    $q->andwhere("e.termination_id IS NULL");
-                }
-                $employeeIds = $q->execute(array(), Doctrine_Core::HYDRATE_SINGLE_SCALAR);
+            $q = Doctrine_Query:: create()
+                ->select('e.empNumber')
+                ->from('Employee e');
 
-                // If only one result, Doctrine_Core::HYDRATE_SINGLE_SCALAR gives a single string.
-                if (is_string($employeeIds)) {
-                    $employeeIds = array($employeeIds);
-                }
-                return $employeeIds;
-        
-        // @codeCoverageIgnoreStart
+            if ($excludeTerminatedEmployees) {
+                $q->andwhere("e.termination_id IS NULL");
+            }
+            $employeeIds = $q->execute(array(), Doctrine_Core::HYDRATE_SINGLE_SCALAR);
+
+            // If only one result, Doctrine_Core::HYDRATE_SINGLE_SCALAR gives a single string.
+            if (is_string($employeeIds)) {
+                $employeeIds = array($employeeIds);
+            }
+            return $employeeIds;
+
+            // @codeCoverageIgnoreStart
         } catch (Exception $e) {
             throw new DaoException($e->getMessage(), $e->getCode(), $e);
         }
         // @codeCoverageIgnoreEnd
     }
-    
+
     /**
      * Get List of Employee Properties
-     * 
+     *
      * @version 2.7.1
      * @param Boolean $excludeTerminatedEmployees
-     * @returns Array List of Employee Properties 
+     * @returns Array List of Employee Properties
      * @throws DaoException
      */
-    public function getEmployeePropertyList($properties, $orderField, $orderBy, $excludeTerminatedEmployees = false) {
+    public function getEmployeePropertyList($properties, $orderField, $orderBy, $excludeTerminatedEmployees = false)
+    {
 
         try {
-                $q = Doctrine_Query :: create();
-                foreach ($properties as $property) {
-                    $q->addSelect($property);
-                }
-                $q->from('Employee e');
-                
-                if ($excludeTerminatedEmployees) {
-                    $q->andwhere("e.termination_id IS NULL");
-                }
-                
-                if ($orderField && $orderBy) {
-                    $orderBy = strcasecmp($orderBy, 'DESC') === 0 ? 'DESC' : 'ASC';
-                    $q->orderBy($orderField . ' ' . $orderBy);
-                }
+            $q = Doctrine_Query:: create();
+            foreach ($properties as $property) {
+                $q->addSelect($property);
+            }
+            $q->from('Employee e');
+
+            if ($excludeTerminatedEmployees) {
+                $q->andwhere("e.termination_id IS NULL");
+            }
+            /** not include purge employees**/
+            $q->andWhere("e.purged_at IS NULL");
```

Comment:
this is the only line I change in EmployeeDao. File was not format wel so I for mat the file.